File: examples/android/MSDKSampleAppJava/lib/src/main/java/msdk/java/handlers/ExperimentalWalletBackup.java
package msdk.java.handlers;

import androidx.annotation.experimental.Experimental;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Retention(RetentionPolicy.CLASS)
@Target({ElementType.METHOD})
@Experimental(level = Experimental.Level.ERROR)
public @interface ExperimentalWalletBackup {
}

File: examples/android/MSDKSampleAppKotlin/app/src/main/java/msdk/kotlin/sample/handlers/ExperimentalWalletBackup.kt
package msdk.kotlin.sample.handlers

import androidx.annotation.experimental.Experimental
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@Retention(RetentionPolicy.CLASS)
@Target(
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER
)
@Experimental(level = Experimental.Level.ERROR)
annotation class ExperimentalWalletBackup